.github/actions/spell-check/expect.txt

```diff
@@ -1800,14 +1418,14 @@ shtypes
 sia
 SIATTRIBFLAGS
 SICHINT
-sid
+SIDs
 siex
 sigdn
 SIGNINGSCENARIO
 Signtool
+sinclairinat
```

Comment:
This is part of a username. Why are we removing from names.txt and adding in expect.txt?